### PR TITLE
Fix Configuration API unsubscribe endpoint 

### PR DESCRIPTION
**Pull request by sarsharma**
Signed-off-by: Sarthak Sharma <sartsharma@microsoft.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add correct endpoint for Configuration API unsubscribe operation.
## Issue reference

#2796

### CODE REVIEW

**Review by akhilac1 — APPROVED** [review]
LGTM

File: daprdocs/content/en/reference/api/configuration_api.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ GET http://localhost:<daprPort>/v1.0-alpha1/configuration/<storename>
 Parameter | Description
 --------- | -----------
 `daprPort` | The Dapr port
-`storename` | The `metadata.name` field component file. Refer to the [component schema] ({{< ref component-schema.md>}})
+`storename` | The `metadata.name` field component file. Refer to the [component schema]({{< ref component-schema.md>}}).
```
Comment (mukundansundar) [inline daprdocs/content/en/reference/api/configuration_api.md]:
```suggestion
`storename` | The `metadata.name` field component file. Refer to the [component schema]({{< ref component-schema.md>}})
```

**Review by mukundansundar — COMMENTED** [review]
Have full stops in all parameters or none of them... Please follow what's there in other docs....

File: daprdocs/content/en/reference/api/configuration_api.md (hunk 1 above)
Comment (sarsharma) [reply daprdocs/content/en/reference/api/configuration_api.md]: updated, thanks!

**Review by mukundansundar — APPROVED** [review]
Lgtm

File: daprdocs/content/en/reference/api/configuration_api.md [hunk 2]
```diff
@@ -165,18 +166,22 @@ None
...
+{
+    "ok" : true
+}
+```
 
 ### Example
 
 ```shell
-curl -X GET 'http://localhost:3500/v1.0-alpha1/configuration/bf3aa454-312d-403c-af95-6dec65058fa2/unsubscribe' 
+curl -X GET 'http://localhost:3500/v1.0-alpha1/configuration/myconfigstore/bf3aa454-312d-403c-af95-6dec65058fa2/unsubscribe'
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/configuration_api.md]: Either suggest changing to all `myconfigstore` or change this example to `mystore`

**Review by msfussell — APPROVED** [review]
LGTM

File: daprdocs/content/en/reference/api/configuration_api.md (hunk 2 above)
Comment (msfussell) [reply daprdocs/content/en/reference/api/configuration_api.md]: I fixed the above to make consistent

Comment (sarsharma) [reply daprdocs/content/en/reference/api/configuration_api.md]: thanks!

**Pull request merged**